website/server/controllers/api-v3/hall.js

```diff
@@ -11,17 +11,112 @@ import _ from 'lodash';
  * Contributors of tier 8 or higher can use this route.
  */
 
+/**
+ * @apiDefine NoAuthHeaders Missing authentication headers
+ *
+ * @apiError (401) {NotAuthorized} NoAuthHeaders Missing authentication headers
+ *
+ * @apiErrorExample Missing authentication headers
+ * {
+ *   "success": false,
+ *   "error": "NotAuthorized",
+ *   "message": "Missing authentication headers."
+ * }
+ */
+
+/**
+* @apiDefine NoAccount There is no account that uses those credentials.
+*
+* @apiError (401) {NotAuthorized} NoAccount There is no account that uses those credentials
+*
+* @apiErrorExample No account
+* {
+*   "success": false,
+*   "error": "NotAuthorized",
+*   "message": "There is no account that uses those credentials."
+* }
+*/
+
+/**
+ * @apiDefine NotAdmin You don't have admin access.
+ *
+ * @apiError (401) {NotAuthorized} NotAdmin User is not an admin
+ *
+ * @apiErrorExample No admin access
+ * {
+ *   "success": false,
+ *   "error": "NotAuthorized",
+ *   "message": "You don't have admin access."
+ * }
+ */
+
+/**
+ * @apiDefine NoUser No user
+ * @apiError (404) {NotFound} NoUser The specified user could not be found.
+ *
+ * @apiErrorExample No user
+ * {
+ *   "success": false,
+ *   "error": "NotFound",
+ *   "message": "User with id \"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx\" not found."
+ * }
+ */
+
 let api = {};
 
 /**
  * @api {get} /api/v3/hall/patrons Get all patrons
- * @apiDescription Only the first 50 patrons are returned. More can be accessed passing ?page=n
+ * @apiDescription Returns an array of objects containing the patrons who backed Habitica's
+ * original kickstarter. The array is sorted by the backer tier in descending order.
+ * By default, only the first 50 patrons are returned. More can be accessed passing ?page=n
  * @apiName GetPatrons
  * @apiGroup Hall
  *
- * @apiParam {Number} page Query Parameter - The result page. Default is 0
+ * @apiHeader {UUID} x-api-user User ID
```

Comment:
We don't need to apply apiHeader to route documentation. It's assumed unless stated that you don't need it.